frontend/src/pulldasher/index.tsx

```diff
@@ -33,9 +33,9 @@ function Pulldasher() {
   const allPulls = useAllPulls();
   const pulls = useAllOpenPulls();
   const pullsCIBlocked = pulls.filter((pull) => pull.isCiBlocked());
-  const pullsDeployBlocked = pulls.filter((pull) => pull.isDeployBlocked());
+  const pullsDeployBlocked = pulls.filter((pull) => pull.isDeployBlocked() || pull.hasMergeConflicts);
   const pullsReady = pulls.filter(
-    (pull) => pull.isReady() && pull.isCiRequired()
+    (pull) => pull.isReady() && pull.isCiRequired() && !pull.hasMergeConflicts()
```

Comment:
I might augment the `isReady()` function to check for merge conflicts instead of adding this each place `isReady()` is used.